Measures/AddDetailedHVAC/measure.xml

```diff
@@ -63,6 +87,132 @@
       <required>true</required>
       <model_dependent>false</model_dependent>
     </argument>
+    <argument>
+      <name>hvacSchedFeiertag</name>
```

Comment:
I think we should change this to Holiday
and likewise Werktag to Weekday and so on

Reply:
Is adressed in a later commit (branch rename-measure-arguments)